src/header/common/range.rs

```diff
@@ -0,0 +1,199 @@
+use std::fmt::{self, Display};
+use std::str::FromStr;
+
+use header::{Header, HeaderFormat, RangeUnit};
+use header::parsing;
+
+/// The `Range` header defined by HTTP/1.0.
```

Comment:
And could you please adjust the doccomment format like in the [Accept](https://github.com/hyperium/hyper/blob/master/src/header/common/accept.rs) header.